elpy.el

```diff
@@ -3633,6 +3633,10 @@ or unless NAME is no callable instance."
      (require 'eldoc)
      (setq eldoc-minor-mode-string nil))
     (`buffer-init
+     ;; avoid eldoc message flickering
+     (when (member 'elpy-module-company elpy-modules)
```

Comment:
This can be `memq` … I'm not sure I really like this condition. Is it harmful to add the following line even if elpy-module-company is disabled?

Reply:
I don't think so.
But in that case maybe we can restrain the number of company function added to `eldoc-message-commands` to avoid side-effects.